runtime/onert/core/include/ir/train/TrainableOperation.h

```diff
@@ -42,10 +42,8 @@ class TrainableOperation : public ITrainableOperation
   virtual bool isRequiredForBackward() const final { return _required_for_backward; }
 
 private:
-  // TODO: Change _trainable and _required_for_backward to false after merge other parts of
-  // fine-tuning feature
-  bool _trainable = true;
-  bool _required_for_backward = true;
+  bool _trainable = false;
+  bool _required_for_backward = false;
```

Comment:
How about making a seperate PR for this? Because it changes overall behavior. This change is easy to roll back. So it is better to make a seperate PR.